eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -333,8 +337,12 @@ function DeleteArmDeployments([object]$ResourceGroup) {
   if (!$DeleteArmDeployments) {
       return
   }
-  Write-Host "Deleting ARM deployments for group $($ResourceGroup.ResourceGroupName) as they may contain secrets. Deployed resources will not be affected."
-  $null = Get-AzResourceGroupDeployment -ResourceGroupName $ResourceGroup.ResourceGroupName | Remove-AzResourceGroupDeployment
+  $toDelete = Get-AzResourceGroupDeployment -ResourceGroupName $ResourceGroup.ResourceGroupName | Where-Object { $_.Outputs.Count }
```

Comment:
Are there possibility of secrets outside of the outputs?

Reply:
They could be on the inputs/parameters but we haven't seen those anywhere. I'm just adding it here because deleting deployments takes quite a while and this narrows the operations by about 90%